Gemfile

```diff
@@ -1,65 +1,62 @@
 source 'http://rubygems.org'
 
-gem 'plek', '1.3.1'
+gem 'plek', '~> 1.11.0'
 
-gem 'rails', '3.2.22'
+gem 'rails', '4.2.4'
 gem 'unicorn', '4.3.1'
 
-gem "mongoid", "~> 3.1.5"
-gem "mongoid_rails_migrations", "~> 1.0.0"
+gem "mongoid", "~> 4.0"
+gem "mongoid_rails_migrations", "~> 1.1.0"
 
 gem 'airbrake', '~> 4.1.0'
 
 gem 'govuk_admin_template', '3.0.0'
-gem 'formtastic', '2.3.0'
-gem 'formtastic-bootstrap', '3.0.0'
+gem 'formtastic', '~> 3.1.2'
+gem 'formtastic-bootstrap', '~> 3.1.1'
 
-gem 'gds-api-adapters', '20.1.1'
+gem 'gds-api-adapters', '~> 24.5.0'
 gem 'statsd-ruby', '1.0.0', :require => 'statsd'
 
 if ENV['BUNDLE_DEV']
   gem 'gds-sso', :path => '../gds-sso'
 else
-  gem 'gds-sso', '9.3.0'
+  gem 'gds-sso', '~> 11.0.0'
 end
 
-gem 'inherited_resources', '~> 1.4.1'
-gem 'whenever'
-gem 'kaminari', '0.14.1'
-gem 'bootstrap-kaminari-views', '0.0.3'
+gem 'responders', '~> 2.0'
+gem 'inherited_resources', '~> 1.6.0'
+gem 'kaminari', '~> 0.16.0'
+gem 'bootstrap-kaminari-views', '~> 0.0.3'
 
-gem 'sidekiq', '2.16.1'
-gem 'sidekiq-delay', '1.0.4'
+gem 'sidekiq', '~> 2.16'
 
-gem 'state_machine', '1.2.0'
+gem 'state_machines-mongoid', '~> 0.1.1'
 gem 'logstasher', '0.4.8'
-gem 'sass-rails', '3.2.6'
 
-group :assets do
-  gem "therubyracer", "~> 0.12.0"
-  gem 'uglifier'
-end
+gem 'sass-rails', '~> 5.0'
+gem "therubyracer", platforms: :ruby
```

Comment:
We can remove this dependency now. Everyone has nodejs installed these days and `ExecJS` does the rest.